10up/ElasticPress
# WooCommerce weight by date

### Description of the Change
This PR will add two options to control the weighting in the search results for products. One will disable the weighting only for the product queries and the other will disable the weighting on any query that includes products.



Closes #3177 

### How to test the Change

- Enable the WooCommerce feature
- Notice that the Weight results by date settings will now have two additional options in it
- Select the third option (Disable for products) and search for a product on the products page(admin) and note that it will not have weighting. Normal search will have weighting even if it contains products in it. 
- Select the 4th option(Disable Weighting in any query that includes products) and search for a product and note that it will not have weighting. Any query that has a post-type product in it will not have the weighting args.
-  Selecting the 2nd option will disable it for any search you make.



### Changelog Entry

> Added - Two new options for Weight results by date to manage weighting for product post-types

### Credits

Props @MARQAS @felipeelia 


### Checklist:
